docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,49 +1,58 @@
 [[run-digger-installer]]
-== Run Digger Installer
+=== Run Digger Installer
 
-.To run the Digger installer, follow these steps:
 
-. Clone the https://github.com/aerogear/digger-installer[Digger Installer GitHub repo^]
+. Clone the https://github.com/aerogear/aerogear-digger-installer[Digger Installer GitHub repo^]
+
+. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure. https://github.com/aerogear/aerogear-digger-installer/blob/master/inventory-sample[Sample Ansible Inventory file^].
```

Comment:
@pmacko1 could we make the link belong to the text `inventory-sample` file rather than a new line?

Reply:
makes sense.
resolved